firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+WebViewDelegates.swift

```diff
@@ -298,23 +298,25 @@ extension BrowserViewController: WKUIDelegate {
                         .successValue ?? false
                     actions.append(isBookmarkedSite ? removeAction : addBookmarkAction)
 
-                    actions.append(UIAction(
-                        title: .ContextMenuDownloadLink,
-                        image: UIImage.templateImageNamed(
-                            StandardImageIdentifiers.Large.download
-                        ),
-                        identifier: UIAction.Identifier("linkContextMenu.download")
-                    ) { _ in
-                        // This checks if download is a blob, if yes, begin blob download process
-                        if !DownloadContentScript.requestBlobDownload(url: url, tab: currentTab) {
-                            // if not a blob, set pendingDownloadWebView and load the request in
-                            // the webview, which will trigger the WKWebView navigationResponse
-                            // delegate function and eventually downloadHelper.open()
-                            self.pendingDownloadWebView = currentTab.webView
-                            let request = URLRequest(url: url)
-                            currentTab.webView?.load(request)
-                        }
-                    })
+                    if url.scheme != "javascript" {
```

Comment:
This seems Ok to me. I'm not entirely clear if there are any scenarios in which we want to support `javascript://` from the Download button. Might be good to get 👀  from @nbhasin2 also, but otherwise LGTM.

Reply:
I missed this but there is a place where we perform safe url. @dicarobinho have you looked into other areas for safe url?

Note: last time we did this was for urlbar where we do not allow for js scheme to run

Reply:
I found it below, basically we make it safe URL for download helper. 

https://github.com/mozilla-mobile/firefox-ios/blob/main/firefox-ios/Client/Frontend/Browser/OpenInHelper/DownloadHelper.swift#L67C6-L67C41

This is okay for now but if we hear any bypasses around this then we will need to further make the check for its absolute string instead of scheme.